packages/react/src/components/InAppMessaging/BannerMessage/styles.css

```diff
@@ -0,0 +1,84 @@
+.body {
+  color: #000;
+  font-size: 16px;
+  font-weight: 400;
+  text-align: left;
+}
+.button {
+  background-color: #e8e8e8;
+  border-radius: 5;
+  border-style: none;
+  width: 100%;
+}
+.buttonsContainer {
```

Comment:
From what I've seen, the UI package tries to opt for BEM conventions. But even disregarding that, CSS conventions usually opt for `kebab-case` rather than JS's `camelCase`